moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/status_codes.h

```diff
@@ -48,14 +48,16 @@ enum StatusCode : int8_t
   NO_WARNING = 0,
   DECELERATE_FOR_SINGULARITY = 1,
   HALT_FOR_SINGULARITY = 2,
-  COLLISION = 3,
-  JOINT_BOUND = 4
+  DECELERATE_FOR_COLLISION = 3,
+  HALT_FOR_COLLISION = 4,
+  JOINT_BOUND = 5
 };
 
 const std::unordered_map<uint, std::string>
     JOG_ARM_STATUS_CODE_MAP({ { NO_WARNING, "No warnings" },
                               { DECELERATE_FOR_SINGULARITY, "Close to a singularity, decelerating" },
                               { HALT_FOR_SINGULARITY, "Very close to a singularity, halting" },
-                              { COLLISION, "Close to a collision, halting." },
+                              { DECELERATE_FOR_COLLISION, "Close to a collision, decelerating." },
+                              { HALT_FOR_COLLISION, "Close to a collision, halting." },
```

Comment:
Shouldn't this be "Collision detected, emergency stop" or something similar?

Reply:
OK